test_scripts/Capabilities/PersistingHMICapabilities/common.lua

```diff
@@ -0,0 +1,419 @@
+          print(key)
+          if not m.isTableEqual(data.payload[key], pCapResponse[key]) then
+            msg = msg .. key.." contains unexpected parameters\n"..
+              " Expected: " .. m.tableToString(pCapResponse[key]) .. "\n" ..
+              " Actual: " .. m.tableToString(data.payload[key]) .. "\n"
+          end
+        end
+        if string.len(msg) > 0 then
+          return false, msg
+        else
+          return true
+        end
+      end)
+      :Do(function()
+          session:ExpectNotification("OnHMIStatus",
+            { hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+          session:ExpectNotification("OnPermissionsChange")
+          :Times(AnyNumber())
+          local policyMode = SDL.buildOptions.extendedPolicy
+          if policyMode == policyModes.P or policyMode == policyModes.EP then
+            session:ExpectNotification("OnSystemRequest", { requestType = "LOCK_SCREEN_ICON_URL" })
+          end
+        end)
+    end)
+end
+
+function m.masterReset(pExpFunc)
+  local isOnSDLCloseSent = false
+  if pExpFunc then pExpFunc() end
+  m.getHMIConnection():SendNotification("BasicCommunication.OnExitAllApplications", { reason = "MASTER_RESET" })
+  m.getHMIConnection():ExpectNotification("BasicCommunication.OnSDLClose")
+  :Do(function()
+    isOnSDLCloseSent = true
+    SDL.DeleteFile()
+  end)
+  :Times(AtMost(1))
+  m.wait(3000)
+  :Do(function()
+    if isOnSDLCloseSent == false then utils.cprint(35, "BC.OnSDLClose was not sent") end
+    if SDL:CheckStatusSDL() == SDL.RUNNING then SDL:StopSDL() end
+  end)
+end
+
+function m.ignitionOff()
+  config.ExitOnCrash = false
+  local timeout = 5000
+  local function removeSessions()
+    for i = 1, m.getAppsCount() do
+      test.mobileSession[i] = nil
+    end
+  end
+  local event = events.Event()
+  event.matches = function(event1, event2) return event1 == event2 end
+  EXPECT_EVENT(event, "SDL shutdown")
+  :Do(function()
+    removeSessions()
+    StopSDL()
+    config.ExitOnCrash = true
+  end)
+  m.getHMIConnection():SendNotification("BasicCommunication.OnExitAllApplications", { reason = "SUSPEND" })
+  m.getHMIConnection():ExpectNotification("BasicCommunication.OnSDLPersistenceComplete")
+  :Do(function()
+    m.getHMIConnection():SendNotification("BasicCommunication.OnExitAllApplications",{ reason = "IGNITION_OFF" })
+    for i = 1, m.getAppsCount() do
+      m.getMobileSession(i):ExpectNotification("OnAppInterfaceUnregistered", { reason = "IGNITION_OFF" })
+    end
+  end)
+  m.getHMIConnection():ExpectNotification("BasicCommunication.OnAppUnregistered", { unexpectedDisconnect = false })
+  :Times(m.getAppsCount())
+  local isSDLShutDownSuccessfully = false
+  m.getHMIConnection():ExpectNotification("BasicCommunication.OnSDLClose")
+  :Do(function()
+    m.print(35, "SDL was shutdown successfully")
+    isSDLShutDownSuccessfully = true
+    RAISE_EVENT(event, event)
```

Comment:
@VjKlepikov Please use `actions.mobile.getConnection():RaiseEvent` instead of `RAISE_EVENT`

Reply:
done